### DISCUSSION TITLE
Link with android and ios part

### DISCUSSION THREAD

**Discussion by pfnts99** [GitLiveApp/firebase-kotlin-sdk#195] [General]
Hello,

I am new to the KMM world and I am making an instant messaging application with Firebase.
I found your github project which seems very interesting, however, I'm a bit stuck.
I've done the installation of firebase on the android and ios side, I've installed the dependencies on the shared module, after that, I don't really know how to configure and link with your project.
Could you help me?
Thanks for your help.

Have a nice day

**Comment by tiagonuneslx** [comment]
Cloud Messaging is not supported yet, you can see in the readme, it has 0% coverage. You could submit an issue requesting this feature https://github.com/GitLiveApp/firebase-kotlin-sdk/issues/new/choose

As for linking your app to this project, all you need to do is to add the dependency you need in commonMain sourceSet:

```Kotlin
 sourceSets {
   val commonMain by getting {
     implementation("dev.gitlive:firebase-firestore:1.3.1")
   }
 }
 ```
 
Of course, dev.gitlive:firebase-messaging:1.3.1 won't be resolved because it doesnt exist yet.

**Reply by pfnts99** [reply]
thanks!
